merlin/loader/torch.py

```diff
@@ -29,7 +29,7 @@
     np.int64: torch.int64,
     np.float16: torch.float16,
     np.float32: torch.float32,
-    np.float64: torch.float64,
+    np.float64: torch.double,
```

Comment:
Since we're using `float16` instead of `torch.half` and `float32` instead of `float`, seems like it makes sense to use `float64` instead of `double` here just for the sake of consistency?